### PR TITLE
feat: new upload dialog

### PR DESCRIPTION
**Pull request by seavan**
copy changes
ability to add message to file shared
helper functions for clients

### CODE REVIEW

**Comment by anri-asaturov** [comment]
Does this PR has corresponding desktop PR?

**Comment by anri-asaturov** [comment]
Clubhouse issue?

**Comment by seavan** [comment]
https://app.clubhouse.io/peerio/story/220/mobile-file-preview-and-review-on-upload-in-chat

it doesn't. desktop is not ready yet.

File: src/models/chats/chat.file-handler.js [hunk 1]
```diff
@@ -71,14 +72,15 @@ class ChatFileHandler {
     /**
      * Shares previously uploaded files to chat.
      * @param {Array<File>} files
+     * @param {string} [message = ''] message to attach to file
      * @returns {Promise}
      */
-    share(files) {
+    share(files, message = '') {
```
Comment (anri-asaturov) [inline src/models/chats/chat.file-handler.js]: This is not taking into consideration the fact that desktop sends rich text format now. (Chat#sendRichTextMessage)

Comment (seavan) [reply src/models/chats/chat.file-handler.js]: there's no rich text editor in FileShare dialog. how should I take it into consideration?

Comment (anri-asaturov) [reply src/models/chats/chat.file-handler.js]: I haven't seen any dialogs, but messages sent from desktop are in rich text format now.

**Comment by anri-asaturov** [comment]
Merging because it's sufficient for mobile, but before we release desktop this needs to be fixed.

**Pull request merged**